src/AuthService.ts

```diff
@@ -10,6 +10,8 @@ export interface AuthServiceProps {
   contentType?: string
   location: Location
   provider: string
+  authorizeEndpoint: string
```

Comment:
Would this make them required parameters? That might break existing code out there.

Reply:
Yes, you're right. This should be optional to not break existing usages.